services/distribution/src/main/resources/application.yaml

```diff
@@ -187,6 +187,10 @@ services:
         height: ${EVREG_QR_CODE_POSTER_TEMPLATE_ANDROID_DESCRIPTION_HEIGHT:20}
         font-size: ${EVREG_QR_CODE_POSTER_TEMPLATE_ANDROID_DESCRIPTION_FONT_SIZE:10}
         font-color: ${EVREG_QR_CODE_POSTER_TEMPLATE_ANDROID_DESCRIPTION_FONT_COLOR:#000000}
+    presence-tracing-parameters:
+      qr-code-error-correction-level: ${EVREG_QR_CODE_ERROR_CORRECTION_LEVEL:0}
+      probability-to-fake-check-ins-if-no-check-ins: ${EVREG_PROBABILITY_TO_FAKE_CHECK_INS_IF_NO_CHECK_INS:0}
```

Comment:
Both probabilities should be part of `plausibleDeniabilityParameters` according to [spec](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/fabad0408daf640168d8197efe15a00abb0d806a/docs/spec/handling-of-app-config.md#parameters-for-event-registration).